doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+
+As we construct the new set of anchors from the last, we need to compute the precisions of the next group to know which one to choose. We can't calculate this directly; instead, we sample from $\mathcal{D}(z|A)$ to obtain an estimate. To do this, we use several different methods for different data types. For instance, in the case of textual data, we want to generate sentences with the anchor words and ensure that they make sense within the data distribution. One option is to replace the missing words (those not in the anchor) from the instance with words with the same POS tag. This means the resulting sample will make semantic sense rather than being a random set of words. Other methods are available. In the case of images, we sample an image from the data set and then superimpose the super-pixels on top of it.
+
+**Pros**
+- Easy to explain as rules are simple to interpret
+- Is a black-box method as we just need to predict the value of an instance and don't need to access model internals
+- Can be parallelized and made to be much more efficient as a result
+- Although they apply to a local instance, the notion of coverage also gives a level of global insight
+
+**Cons**
+- This algorithm is much slower for high dimensional feature spaces
+- If choosing an anchor close to a decision boundary, the method may require a considerable number of samples from $\mathcal{D}(z|A)$ and $\mathcal{D}(z|A')$ to distinguish two anchors $A$ and $A'$.
+- High dimensional feature spaces such as images need to be reduced. Typically, this is done by segmenting the image into superpixels. The choice of the algorithm used to obtain the superpixels has an effect on the anchor obtained.
+- Practitioners need to make several choices concerning parameters and domain-specific setup. For instance, the precision threshold or the method by which we sample $\mathcal{D}(z|A)$. Fortunately, alibi provides default settings for a lot of specific data types.
+
+
+#### Pertinent Positives
+
+| Model-types | Task-types     | Data-types  |
+| ----------- | -------------- | ----------- |
+| Black-box   | Classification | Tabular     |
+|             |                | Image       |
+
+
+Informally a Pertinent Positive is the subset of an instance that still obtains the same classification. These differ from anchors primarily in the fact that they aren't constructed to maximize coverage. The method to create them is also substantially different. The rough idea is to define an absence of a feature and then perturb the instance to take away as much information as possible while still retaining the original classification.
```

Comment:
- "subset of an instance" is probably too informal, better "subset of features of an instance" ?
 - Depending on what we do with coverage in the Anchors section may need to remove all references to it
 - emphasize *absence of a feature*